pgjdbc/src/main/java/org/postgresql/core/ParameterContext.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2020, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.core;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * org.postgresql.core.Parser stores information about placeholder occurrences in ParameterContext.
+ * In case of standard JDBC placeholders ('?') the position in the SQL text is recorded.
+ * For named placeholders (':paramName') the name is recorded as well as the position.
+ * PgPreparedStatement can then use the name to lookup the parameter corresponding index.
```

Comment:
use either @link or @code around PgPreparedStatement

Reply:
I went with code as link doesn't seem to work for non-public classes